subsys/bluetooth/host/classic/shell/bredr.c

```diff
@@ -64,8 +71,10 @@ static int cmd_auth_pincode(const struct shell *sh,
 	}
 
 	if (strlen(argv[1]) > max) {
-		shell_print(sh, "PIN code value invalid - enter max %u "
-			    "digits", max);
+		shell_print(sh,
+			    "PIN code value invalid - enter max %u "
+			    "digits",
```

Comment:
This looks like it should fit on a single line

Reply:
Let me check it. It is formatted by clang-format.

Reply:
Updated.